CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.0.0)
 
 include (utils.cmake)
 
 disallow_intree_builds()
 
-project (utf8proc C)
+if (POLICY CMP0048)
+  cmake_policy (SET CMP0048 NEW)
+endif ()
+project (utf8proc VERSION 2.6.1 LANGUAGES C)
 
 # This is the ABI version number, which may differ from the
 # API version number (defined in utf8proc.h).
```

Comment:
Maybe add a comment here to remind us that the API version number should also be set in the `project` statement above when it is updated in utf8proc.h.

Reply:
> Maybe add a comment here to remind us that the API version number should also be set in the `project` statement above when it is updated in utf8proc.h.

Might be a good idea - please merge this and then adapt to your liking.